guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -41,26 +41,16 @@ endif::[]
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
-. Optional: Select a different *Organization*.
-. Optional: Select a different *Location*.
-. Optional: From the *Host Group* list, select the host group to associate the hosts with.
-Fields that inherit value from *Host group*: *Operating system*, *Activation Keys* and *Lifecycle environment*.
-+
-If your host group has any Ansible roles assigned, the Ansible roles will run on your host upon the registration.
-. Optional: From the *Operating system* list, select the operating system of hosts that you want to register.
-ifndef::satellite,orcharhino[]
-Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
-endif::[]
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
-+
-[NOTE]
-====
-A {SmartProxy} behind a load balancer takes precedence over a {SmartProxy} selected in the {ProjectWebUI} as the host's content source.
-====
-ifdef::katello,satellite,orcharhino[]
-. In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
-endif::[]
-. Optional: Select the *Insecure* option, if you want to make the first call insecure.
+. Enter the details for how you want the registered hosts to be configured.
```

Comment:
The mandatory fields are marked with an asterisk:

![Screenshot from 2024-10-01 11-30-33](https://github.com/user-attachments/assets/839b78ba-d22c-428f-adad-c8772964a707)

I would argue that this is enough and there is no need to duplicate the same information in the guide.

Brevity is one reason to do it this way. Another is maintainability: We wouldn't need to worry about making sure we list all the mandatory fields -- the web UI already does!

Does this address your concern?

Reply:
I still think it's better to document a mandatory field even when it is marked in the UI. It's a single step, so brevity is not a concern in this case. 

In my experience, maintainability is not a major concern because mandatory fields tend not to change. (Same cannot be said about options.) Sometimes a writer forgets to update prerequisites. If a step says "Enter the activation keys.", this alerts the user to a possible prerequisite. 

I tend to read through procedures before performing them and I like to know what I'm doing before I start. That's my personal preference. Other users might not feel that way. 

As I said, it is your call.

Reply:
That makes sense, thanks for explaining your point of view. I'll add it.